GTG/gtk/editor/editor.py

```diff
@@ -65,12 +67,12 @@ def __init__(self,
         self.deletebutton = self.builder.get_object("delete_editor")
         self.deletebutton.set_tooltip_text(GnomeConfig.DELETE_TOOLTIP)
         self.subtask_button = self.builder.get_object("insert_subtask")
-        self.subtask_button.set_tooltip_text(GnomeConfig.SUBTASK_TOOLTIP)
-        self.inserttag_button = self.builder.get_object("inserttag")
-        self.inserttag_button.set_tooltip_text(GnomeConfig.TAG_TOOLTIP)
-        self.open_parents_button = self.builder.get_object("open_parents")
-        self.open_parents_button.set_tooltip_text(
-            GnomeConfig.OPEN_PARENT_TOOLTIP)
+        # self.subtask_button.set_tooltip_text(GnomeConfig.SUBTASK_TOOLTIP)
```

Comment:
I don't really know if that's a good idea. The reason for having it like this is that later in the code, we are changing the tooltips based on whether your task is among active, dismissed or marked as done tasks, you know... and therefore it might as well be a matter of convenience to have it like this, done in one way. 

What do you say?